docker-compose.yaml

```
@@ -1,8 +1,8 @@
 services:
   # broker for celery
-  rabbitmq:
+  docker.io/rabbitmq:
```

This can remain the same
```suggestion
  rabbitmq:
```